packages/api/src/index.ts

```diff
@@ -16,6 +17,7 @@ import { logger } from './util/logger';
 
 const env = container.resolve(Env);
 container.register(PrismaClient, { useValue: new PrismaClient() });
+container.register(REST, { useValue: new REST({ version: '9' }).setToken(env.discordToken) });
```

Comment:
leave it as is; we want v10

```suggestion
container.register(REST, { useValue: new REST().setToken(env.discordToken) });
```

Reply:
Done in ffe7ae6